samples/stock/chart/panning/demo.html

```diff
@@ -0,0 +1,5 @@
+<div id="container" style="height: 400px; min-width: 600px"></div>
+
```

Comment:
Add a description to instruct the user to use the `Alt` key for panning. With the current demo, I didn't know that until I read the code.

Reply:
Description added as chart's subtitle.